lxljs/display.js

```diff
@@ -414,7 +444,10 @@ export function getDisplayObject(item, level, resources, quoted, settings) {
             }
           }
         }
-        lxlLog(`Computed alternateProperties for ${trueItem['@type']}. Looked for: ${property.alternateProperties.join(', ')} | Settled on: ${foundProperty}`);
+        // FIXME: Why did I start getting "TypeError: JSON.stringify(...) is undefined" here ??
```

Comment:
Was that the exact error message? Otherwise it might have been due to  `foundProperty` being being left unset (i.e. still `undefined`) in certain cases. That would look like:
```javascript
> JSON.stringify(foundProperty).replaceAll('"', '')
Uncaught TypeError: Cannot read properties of undefined (reading 'replaceAll')
```

Reply:
That was the exact message in firefox, I interpreted it as `JSON.stringify` being undefined... :woozy_face: